src/malloc_interceptor.c

```diff
@@ -196,7 +164,7 @@ static void build_paths(char out_tmp[512], char out_csv[512]) {
         memcpy(base, env_path, n);
         base[n] = '\0';
     } else {
-        snprintf(base, sizeof(base), "/tmp/peak_memlog");
+        snprintf(base, sizeof(base), "./peak_memlog");
```

Comment:
What is the use of `peak_memlog`? Not sure if we should have a standard location like this. Maybe this should also be a user defined parameter?